container-storage-setup.1

```diff
@@ -86,10 +86,16 @@ functionality to work properly.
 \f[B]Supported options for the configuration file\f[]:
 
 STORAGE_DRIVER:
-      Specify a storage driver to be used with container runtime.
-      Default: "devicemapper".
-      Valid values are overlay, overlay2 and "".
-      "" tells container-storage-setup to not perform any storage setup.
+      Specify a storage driver to be used with container runtime; the default
+      value will chosen by your OS vendor.
+      Known values are: devicemapper, overlay, auto, and "".  The empty string ""
+      is used when you want to ensure container-storage-setup does not automatically
+      change configuration (for example, you want to do it manually).  The "auto" option
+      may be used when c-s-s is started by default (for example, Atomic Host);
+      when specified, c-s-s will choose a backend.  Currently, this will be "overlay" if
+      supported by the kernel/filesystem at the target mount point, and if not, then
+      "devicemapper" will be used.
+      Note the "overlay2" option is an alias for "overlay".
```

Comment:
In docker, overlay2 and overlay are two different drivers (and overlay2 is not an alias for overlay).

Reply:
Oh man...yes.  I had thought for some reason we were canonicalizing now but in fact you're right - that would have been a major screwup!